pkg/daemon/ceph/config/config.go

```diff
@@ -54,6 +54,8 @@ type GlobalConfig struct {
 	ClusterAddr              string `ini:"cluster addr,omitempty"`
 	ClusterNetwork           string `ini:"cluster network,omitempty"`
 	MonKeyValueDb            string `ini:"mon keyvaluedb"`
+	LogToStderr		 bool	`ini:"log_to_stderr"`
```

Comment:
Could you please look into running `go fmt` to correctly format this file again.